src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,15 +1,31 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+    private double bladeWidth;
```

Comment:
You should never want to expose the object fields directly. They should be accessed through special methods (getters and setters). Consider adding private access modifier to the 'bladeWidth' field.